bundler/lib/bundler/rubygems_ext.rb

```diff
@@ -221,6 +221,10 @@ class Platform
     MSWIN64 = Gem::Platform.new("mswin64") unless defined?(MSWIN64)
     MINGW = Gem::Platform.new("x86-mingw32") unless defined?(MINGW)
     X64_MINGW = Gem::Platform.new("x64-mingw32") unless defined?(X64_MINGW)
+
+    if RUBY_ENGINE == "truffleruby" && !defined?(REUSE_AS_BINARY_ON_TRUFFLERUBY)
+      REUSE_BINARY_ON_TRUFFLERUY = %w[libv8 sorbet-static]
```

Comment:
It's `REUSE_AS_BINARY_ON_TRUFFLERUBY` not `REUSE_BINARY_ON_TRUFFLERUY`.
I'd prefer if we don't depend on that constant name going forward though.

Reply:
(still a typo here, missing a B before the last Y)